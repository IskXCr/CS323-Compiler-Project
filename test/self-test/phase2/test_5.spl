struct ib
{
    char cn;
};

struct website
{
    struct ib itbill;
    char is;
};

void welcome(int to, float my)
{
    struct website www[10];
    www[to].is = www[my].itbill.cn;
    return my;
}