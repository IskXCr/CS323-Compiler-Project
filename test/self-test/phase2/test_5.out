Error type 12 at line 15: array indexing with a non-integer type expression
Error typy 8 at line 16: incompatible return type